library/src/main/java/com/schibsted/spain/barista/BaristaRecyclerViewActions.java

```diff
@@ -15,7 +14,7 @@
 
   public static void clickRecyclerViewItem(@IdRes int recyclerViewId, int... positions) {
     if (positions.length == 0) {
-      throw new BaristaException("positions cannot be empty");
+      throw new IllegalArgumentException("positions cannot be empty");
```

Comment:
Previously, we wrote "Positions" with Capital letter. Let's maintain just one writing style.

Reply:
Humm according to the diff it was lowercase before. But you're right, we should use capital letters for correctness.

Reply:
+1

Reply:
Oops! I thought I already did it!
Sorry @rocboronat and thanks for the reminder ❤️

Reply:
No prob bro! 💋